pkg/resources/aws/aws.go

```diff
@@ -1339,7 +1339,13 @@ func FindNatGateways(cloud fi.Cloud, routeTables map[string]*resources.Resource,
 			natGatewayId := aws.StringValue(ngw.NatGatewayId)
 
 			forceShared := !ownedNatGatewayIds.Has(natGatewayId)
-			resourceTrackers = append(resourceTrackers, buildNatGatewayResource(ngw, forceShared, clusterName))
+			ngwResource := buildNatGatewayResource(ngw, forceShared, clusterName)
+			resourceTrackers = append(resourceTrackers, ngwResource)
```

Comment:
did you test this? Now you are skipping only elastic ip deletion? Are you sure that nat gateway is not going to be deleted

Reply:
> did you test this? Now you are skipping only elastic ip deletion? Are you sure that nat gateway is not going to be deleted

Yes, because the nat gateway is correctly tagged with ownership tag. I tested both scenarios:
1: Nat gateway owned: nat gateway deleted, elastic IP deleted
2: Nat gateway shared: nat gateway not deleted, elastic IP not deleted